### ISSUE TITLE
prepare for publish

### ISSUE THREAD

**Issue by jonotron** [TwoStoryRobot/react-search-lunr#1]
There are several things I need to finish before this gets published:

* [x] setup build process
* [x] hook up renovate
* [x] setup CI with semaphore
* [ ] setup static site with link to static storybook
* [x] publish to npm

**Comment by jonotron** [comment]
This is essentially done now. Will open a separate issue for static storybook build
